android/app/src/main/java/app/organicmaps/widget/placepage/ElevationProfileViewRenderer.java

```diff
@@ -78,6 +46,32 @@ public void initialize(@Nullable View view)
     mDifficultyLevels[2] = mDifficultyContainer.findViewById(R.id.difficulty_level_3);
   }
 
+  public void render(@Nullable Track track, @NonNull ElevationInfo elevationInfo, @NonNull TrackStatistics stats)
+  {
+    mChartController.setData(track, elevationInfo, stats);
+    setDifficulty(elevationInfo.getDifficulty());
+    mAscent.setText(formatDistance((int) stats.getAscent()));
+    mDescent.setText(formatDistance((int) stats.getDescent()));
+    mMaxAltitude.setText(formatDistance(stats.getMaxElevation()));
+    mMinAltitude.setText(formatDistance(stats.getMinElevation()));
```

Comment:
We have ```Framework.nativeFormatAltitude``` below.
In core we have Distance::FormatAltitude

Reply:
And?